spec/services/idv/steps/verify_step_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe Idv::Steps::VerifyStep do
```

Comment:
I didn't see any previous methods for testing steps outside of feature specs, so I cobbled this together, but it feels a bit weird. Definitely open to suggestions on this.

Reply:
I'm a fan of having unit test like things for these steps, so big +1 from me